src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -53,8 +53,11 @@ public EditCommand parse(String args) throws ParseException {
         if (argMultimap.getValue(PREFIX_REMARK).isPresent()) {
             editStudentDescriptor.setRemark(ParserUtil.parseRemark(argMultimap.getValue(PREFIX_REMARK).get()));
         }
+        if (argMultimap.getValue(PREFIX_TAG).isPresent()) {
+            editStudentDescriptor.setTag(ParserUtil.parseTag(argMultimap.getValue(PREFIX_TAG).get()));
+        }
 
-        parseTagsForEdit(argMultimap.getAllValues(PREFIX_TAG)).ifPresent(editStudentDescriptor::setTags);
+        //parseTagsForEdit(argMultimap.getAllValues(PREFIX_TAG)).ifPresent(editStudentDescriptor::setTag);
```

Comment:
aside from the commented out code question brought up earlier, is parsing tags for edit no longer necessary?